Bugzilla/Search.pm

```diff
@@ -2573,6 +2519,15 @@ sub _triage_owner_pronoun {
   }
 }
 
+sub _get_user_id {
+  my ($self, $value) = @_;
+
+  if ($value =~ /^%\w+%$/) {
```

Comment:
Nit: a comment to clarify what this is would be nice, or maybe put the logic in a well-named function so this becomes `if (value_is_pronoun($value))` or something like that.